test/extended/image_ecosystem/plugin.go

```diff
@@ -565,42 +584,44 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 
 		jenkinsPod := findJenkinsPod(oc)
 
-		ticker = time.NewTicker(10 * time.Second)
-		go func() {
-			for t := range ticker.C {
-				memstats, err := oc.Run("exec").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "--", "cat", "/sys/fs/cgroup/memory/memory.stat").Output()
-				if err != nil {
-					ginkgolog("\nUnable to acquire Jenkins cgroup memory.stat")
-				}
-				ps, err := oc.Run("exec").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "--", "ps", "faux").Output()
-				if err != nil {
-					ginkgolog("\nUnable to acquire Jenkins ps information")
-				}
-				ginkgolog("\nJenkins memory statistics at %v\n%s\n%s\n\n", t, ps, memstats)
-
-				// This is likely a temporary measure in place to extract diagnostic information during unexpectedly
-				// high memory utilization within the Jenkins image. If Jenkins is using
-				// a large amount of RSS, extract JVM information from the pod.
-				if memoryOverragePattern.MatchString(memstats) {
-					histogram, err := oc.Run("rsh").Args("--namespace", jenkinsNamespace, jenkinsPod.Name, "jmap", "-histo", "1").Output()
-					if err == nil {
-						ginkgolog("\n\nJenkins histogram:\n%s\n\n", histogram)
-					} else {
-						ginkgolog("Unable to acquire Jenkins histogram: %v", err)
+		if os.Getenv(disableJenkinsMemoryStats) == "" {
```

Comment:
The file diff isn't ideal here .... if you append `?w=1` to the url you'll see I just wrapped the ticker and go func with the `if os.Getenv...` call